TestON/drivers/common/cli/emulator/remotemininetdriver.py

```diff
@@ -376,9 +376,20 @@ def disconnect( self ):
         if self.handle:
             # Close the ssh connection
             self.handle.sendline( "" )
-            self.handle.expect( "\$" )
-            self.handle.sendline( "exit" )
-            self.handle.expect( "closed" )
+            #self.handle.expect( "\$" )
+            i = self.handle.expect( [ '\$', 'mininet>', pexpect.TIMEOUT ],
```

Comment:
The connect function gives the shell prompt. However, I am utilizing this driver to create a handle from lincoe and run a python script of mininet topology from the shell prompt. In the course, I am dropped to the mininet prompt and so I added an option in the disconnect() to exit from the mininet prompt as well if it was  at the mininet prompt.